assets/js/src/listing/notices.jsx

```diff
@@ -0,0 +1,111 @@
+import MailPoet from 'mailpoet';
+import PropTypes from 'prop-types';
+import React, { Fragment } from 'react';
+import ReactStringReplace from 'react-string-replace';
+
+const resumeMailerSending = () => {
+  MailPoet.Ajax.post({
+    api_version: window.mailpoet_api_version,
+    endpoint: 'mailer',
+    action: 'resumeSending',
+  }).done(() => {
+    MailPoet.Notice.success(MailPoet.I18n.t('mailerSendingResumedNotice'));
+    window.mailpoet_listing.forceUpdate();
+  }).fail((response) => {
+    if (response.errors.length > 0) {
+      MailPoet.Notice.error(
+        response.errors.map(error => error.message),
+        { scroll: true }
+      );
+    }
+  });
+};
+
+function MailerError(props) {
+  if (props.mta_log.error && props.mta_log.status === 'paused' && props.mta_log.error.operation !== 'authorization') {
+    if (props.mta_log.error.operation === 'migration') {
+      return (
+        <div className="mailpoet_notice notice notice-warning">
+          <p>{ props.mta_log.error.error_message }</p>
+        </div>
+      );
+    }
+
+    let message = props.mta_log.error.error_message;
+    const code = props.mta_log.error.error_code;
+    if (code) {
+      message += message ? ', ' : '';
+      message += MailPoet.I18n.t('mailerErrorCode').replace('%$1s', code);
+    }
+    return (
+      <div className="mailpoet_notice notice notice-error">
+        <p>
+          {
+            props.mta_log.error.operation === 'send'
+              ? MailPoet.I18n.t('mailerSendErrorNotice').replace('%$1s', props.mta_method)
+              : MailPoet.I18n.t('mailerConnectionErrorNotice')
+          }
+          :
+          {' '}
+          <i>{ message }</i>
+        </p>
+        { props.mta_method === 'PHPMail' ? <PHPMailerCheckSettingsNotice /> : <MailerCheckSettingsNotice /> }
+        <p>
+          <a
+            href="javascript:;"
+            className="button button-primary"
+            onClick={resumeMailerSending}
+          >
+            { MailPoet.I18n.t('mailerResumeSendingButton') }
+          </a>
+        </p>
+      </div>
+    );
+  }
+  return null;
+}
+
+MailerError.propTypes = {
+  mta_method: PropTypes.string.isRequired,
+  mta_log: PropTypes.object.isRequired, // eslint-disable-line react/forbid-prop-types
+};
+
+const PHPMailerCheckSettingsNotice = () => (
+  <Fragment>
```

Comment:
Why don't you keep `<>`?

Reply:
My syntax highlighter was not happy with it, so I changed it for the dev time, then forgot to bring it back. I will edit that.